sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -91,6 +91,7 @@ export interface FeatureFlagValue {
         }[];
     };
     description?: string;
+    displayName?: string;
```

Comment:
So this field will eventually be used in the portal? (And presumably someone writing their own app config viewer could use it as well)

Reply:
yes, yess!